obs_youtube_uploader/preview/gestures.py

```
@@ -0,0 +1,167 @@
+    if len(code) == 4 and code.startswith("Key"):
+        return code[3].upper()
+    if len(code) == 6 and code.startswith("Digit"):
+        return code[5]
+    if code.startswith("Numpad") and len(code) == 7 and code[6:].isdigit():
+        return code
+    if code.startswith("F") and code[1:].isdigit() and 1 <= int(code[1:]) <= 24:
+        return code
+    return None
+
+
+def _vk(token: str):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_vk`

(ANN202)